test/functional/outputs/kafka/forward_to_kafka_test.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/openshift/cluster-logging-operator/test/helpers/kafka"
 )
 
-var _ = Describe("[Functional][Outputs][Kafka] Functional tests", func() {
+var _ = Describe("[Functional][OutputConditions][Kafka] Functional tests", func() {
```

Comment:
I think this needs to be rolled back. Similarly the other `[OutputConditions]` strings in the tests?